### ISSUE TITLE
BetterMC2 [FABRIC] Modpack not running

### ISSUE THREAD

**Issue by Vartroc** [Infinidoge/nix-minecraft#88]
## Pretext:
I have the BetterMC modpack imported using `packwiz curseforge import PATH` and then did `packwiz mr export` in order to find out all the mods that packwiz couldn't download (It shows a list of the ones it couldn't download). Then I manually removed all the mods that couldn't be downloaded and uploaded the pack to github. Then I added the correct link and tried running it. But when the server starts, it gets a crashing loop and stops at the 5th crash (normal behaviour). I tried the packwiz modpack in a singleplayer world and everything worked fine...

## Questions:
Can I see the crashlogs anywhere? and
Do you have any idea why it crashes?

**Comment by Infinidoge** [comment]
Crash logs are loaded at `/srv/minecraft/servername`, along with the rest of the server files. I don't have any information about why it might be crashing until I get crash logs.

**Comment by Vartroc** [comment]
Of course, here are all the files (except .fabric, because there are the .jars from the mods and that would conflict with many licenses of the mods...)

https://github.com/Vartroc/hexagonal_planar_6
https://paste.gg/p/anonymous/a596befc24da488381762c87a106298a - latest.log (is included in git repo, but I created the paste before making the repo, so maybe it's some use to you idk)
 https://paste.gg/p/anonymous/f516acc3bdbe4eea9599466c70ac61d7 - output of `tree .fabric`

**Comment by Infinidoge** [comment]
Logs 1 and 2 indicate that `configureddefaults` is the one causing the crash. Logs 3, 4, and latest seem to be fine(?)
